pkg/monitor/sqsevent/spot-itn-event.go

```diff
@@ -69,7 +69,7 @@ func (m SQSMonitor) spotITNTerminationToInterruptionEvent(event *EventBridgeEven
 		IsManaged:            nodeInfo.IsManaged,
 		InstanceID:           spotInterruptionDetail.InstanceID,
 		ProviderID:           nodeInfo.ProviderID,
-		Description:          fmt.Sprintf("Spot Interruption event received. Instance %s will be interrupted at %s \n", spotInterruptionDetail.InstanceID, event.getTime()),
+		Description:          fmt.Sprintf("Spot Interruption event received. Instance %s received interruption at %s \n", spotInterruptionDetail.InstanceID, event.getTime()),
```

Comment:
This might still be misinterpreted as "the instance was interrupted at".  How about the following?

```suggestion
		Description:          fmt.Sprintf("Spot Interruption notice for instance %s was sent at %s \n", spotInterruptionDetail.InstanceID, event.getTime()),
```

Reply:
Yes, this looks good to me! Will modify this one! Thanks for the suggestion :)